doc/sdm_appliance.md

```diff
@@ -26,19 +26,19 @@ Depicted below are 2 use-case scenario:
 
 1. Ubuntu 22.03 LTS
 
-2. Intel&reg; Media Transport Library (Intel&reg; MTL codenamed Kahawai)
+2. Media Transport Library (MTL codenamed Kahawai)
 
-3. FFmpeg with kahawai encoder and decoder [patches](https://google.com)
+3. FFmpeg with Kahawai encoder and decoder [patches](https://google.com)
 
 ## 2. Installation and configuration
 
 The demo currently works only on Linux. Follow the steps below to install all the software components required for the demo.
 
-### Build Intel&reg; MTL
+### Build Media Transport Library (MTL)
 
-- See [build.md](./build.md) to build libmtl on linux.
+- See [build.md](./build.md) to build libmtl on Linux.
 
-### Build ffmpeg (with kahawai encoder and decoder)
+### Build ffmpeg (with Kahawai encoder and decoder)
```

Comment:
Why kahawai ?

Reply:
Man, I just made capital letter here. :) Should I replace this word? What should be the pattern of such replacement?